lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -11,15 +11,25 @@ def self.email(params = {}, _secrets = {}, _context = {})
         end
 
         private_class_method def self.email_status!(runner_context)
+          miq_task_status!(runner_context)
+        end
+
+        private_class_method def self.miq_task_status!(runner_context)
           miq_task = ::MiqTask.find(runner_context["miq_task_id"])
-          return if miq_task.nil?
+          return error_payload(:cause => "Unable to find MiqTask id: [#{runner_context["miq_task_id"]}]") if miq_task.nil?
 
           runner_context["running"] = miq_task.state != ::MiqTask::STATE_FINISHED
-          runner_context["success"] = miq_task.status == ::MiqTask::STATUS_OK unless runner_context["running"]
+
+          unless runner_context["running"]
+            runner_context["success"] = miq_task.status == ::MiqTask::STATUS_OK
+            runner_context["output"]  = runner_context["success"] ? miq_task.message : {"Error" => "States.TaskFailed", "Cause" => miq_task.message}
+          end
```

Comment:
do we want to merge this with `error_payload`?

Reply:
Yeah possibly, I considered doing `runner_context = error_payload...` but that would overwrite rather than merge with the argument passed in